.git-blame-ignore-revs

```diff
@@ -1,3 +1,2 @@
-# black
-da7794207585c235994b97b398ace1d7de51f735
```

Comment:
I think this file can contain multiple commit hashes, so let's keep the old one as well